Source/WebCore/css/FontFace.cpp

```diff
@@ -68,7 +69,7 @@ Ref<FontFace> FontFace::create(ScriptExecutionContext& context, const String& fa
         return result;
     }
 
-    bool allowDownloading = context.settingsValues().downloadableBinaryFontsEnabled;
+    bool allowDownloading = context.settingsValues().downloadableBinaryFontAllowedTypes == DownloadableBinaryFontAllowedTypes::Any;
```

Comment:
Should this be:

`context.settingsValues().downloadableBinaryFontAllowedTypes != DownloadableBinaryFontAllowedTypes::None;`

Reply:
You are right, this is the place where we have to do the font hash check for the FontFace api case. I'll move the reusable code from CachedFont to AllowedFonts and add the check here.